### DISCUSSION TITLE
About OG packets and STOP packets

### DISCUSSION THREAD

**Discussion by ChinaGuCheng** [danielpaulus/quicktime_video_hack#132] [General]
After receiving the OG packet, it means that the audio starts to transmit
After receiving the STOP packet, it means that the audio stops transmission, and the audio clock needs to be initialized

Generally, if you turn on the microphone, you will receive STOP, and when you turn off the microphone, you will receive OG.
@danielpaulus

**Comment by ChinaGuCheng** [comment]
![OG后EAT](https://user-images.githubusercontent.com/34159354/183790567-65c87f38-dd23-48aa-ad51-688a0504d7a3.png)
![STOP后没有EAT](https://user-images.githubusercontent.com/34159354/183790575-600ddf65-f389-494c-b205-972422753da5.png)

**Reply by danielpaulus** [reply]
makes a lot of sense as !OG == GO! 
Thanks for clarifying this!
